go.mod

```diff
@@ -13,8 +13,8 @@ require (
 	github.com/fatih/color v1.18.0
 	github.com/google/go-cmp v0.7.0
 	github.com/kong/go-apiops v0.1.43
-	github.com/kong/go-database-reconciler v1.22.7
-	github.com/kong/go-kong v0.65.1
+	github.com/kong/go-database-reconciler v1.22.8-0.20250526131858-6a3a98c9a366
```

Comment:
To be updated post release.
Waiting for this PR merge: https://github.com/Kong/go-database-reconciler/pull/285